managedfile: change upload directory to /var/cache/labgrid

Emantor opened the pull request:
**Description**
Move the cache directory for managedfile uploads to `/var/cache/labgrid` and also provide a configuration example in contrib.
<!---
This checklist roughly outlines the steps for new features, remove and add tasks as needed:
--->
**Checklist**
- [x] Documentation for the feature
- [x] CHANGES.rst has been updated
- [x] PR has been tested

jluebbe commented on CHANGES.rst:
```diff
@@ -1,3 +1,12 @@
+
+Breaking changes in 0.3.0
+~~~~~~~~~~~~~~~~~~~~~~~~~
+- `ManagedFile` now saves the files on a different directory on the exporter.
```
s/on/in/ and also mention the paths.

jluebbe commented on contrib/systemd/tmpfiles.d/labgrid.conf:
```diff
@@ -0,0 +1,7 @@
+# environment
+# Path			Mode	UID	GID	Age Argument
+d /var/cache/labgrid	1775	root	users	-
+e /var/cache/labgrid/*	-	-	-	30d
```
Use a shorter time. Perhaps 2 days?

Emantor commented:
I fixed both your change requests and added a recommendation for `fs.protected_regular=1` and `fs.protected_fifos=1` to the documentation and `CHANGES.rst`.
